.gitignore

```diff
@@ -27,3 +27,100 @@ yarn-error.log*
 
 # lighthouse report
 /localhost_*.report.html
+
+# Jetbrains files
+.idea/
+
+# Local Python virtualenv
+.venv
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Python Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache
+
+# Translations
+*.mo
+*.pot
+
+# pyenv
+.python-version
+
+# celery beat schedule file
+celerybeat-schedule
+
+# SageMath parsed files
+*.sage.py
+
+# Environments
+.env
```

Comment:
`.env` is intentionally allowed, use `.env.local` for stuff outside version control. that said we don't use the environment files for the JS on this project anymore so if you want to leave this you can. if you do leave it maybe combine it with all the env related stuff above

Reply:
Ah, that's a good practice. I'll have to start doing that with the other projects we have. 👍